packages/ui-components/src/components/toggle-tip/toggle-tip.types.ts

```diff
@@ -0,0 +1,14 @@
+export enum EToggleTipPosition {
```

Comment:
Isn't this the same as `ETooltipPosition`?

Reply:
Since we're using the FloatingUI lib to position our elements relatively could we define this enum in the `packages/ui-components/src/utils/types/components.ts` types file?